lib/foreman_tasks/engine.rb

```diff
@@ -69,6 +69,8 @@ class Engine < ::Rails::Engine
 
       config.to_prepare do
         ::Api::V2::HostsController.send :include, ForemanTasks::Concerns::HostsControllerExtension
+        ::PuppetclassesController.send :include, ForemanTasks::Concerns::EnvironmentsExtension
```

Comment:
well, technically, the definition is at Foreman::Controller::Environments.

I could include it there, but this seems to be more verbose...